tests/cases/conformance/es2018/dynamicImport/importCallExpression2ES2018.ts

```diff
@@ -0,0 +1,17 @@
+﻿// @module: es2018
+// @target: esnext
+// @filename: 0.ts
+export class B {
+    print() { return "I am B"}
+}
+
+// @filename: 2.ts
+// We use Promise<any> for now as there is no way to specify shape of module object

```

Comment:
So being able to represent the shape of import module object is the second half of this feature.... I have something that I will put it soon (e.g possibly new syntax to describe shape so you don't have to do namespace import). Sidenote Typescript does import elision if not use as value